src/components/Navbar/Navbar.tsx

```diff
@@ -64,9 +36,9 @@ export default function Navbar() {
               href="https://bit.ly/joinreactdevske"
               rel="noopener noreferrer"
             >
-              <button className="flex mx-auto bg-gray-300 border-0 py-[10px] px-[33px] focus:outline-none hover:bg-gray-600 hover:text-white rounded text-lg">
+              <Button className="bg-[#EC0505] w-[203px] h-[46px] rounded-md text-base text-white font-montserrat font-bold">
                 Join Community
-              </button>
+              </Button>
             </a>
```

Comment:
Wrapping the button inside the link creates two nested tabbable elements, we should get rid of the `a` and use the `LinkButton` instead of the normal `Button`.

Reply:
Fixed via [102c9e6](https://github.com/reactdeveloperske/reactdevske-website/pull/105/commits/102c9e6d96fd1bdbe13da1a8e6343a327a357a77)